src/useFlag.ts

```diff
@@ -21,6 +23,11 @@ const useFlag = (name: string) => {
       const enabled = isEnabled(name);
       setFlag(enabled);
     });
+
+    return () => {
+      client.off('update');
+      client.off('ready');
```

Comment:
Should we pass in the handlers here, to match the `on` calls above? Same for `useVariant`.

Reply:
Yeah I was a bit unsure, but the tinyEmitter library says you can omit these: https://www.npmjs.com/package/tiny-emitter

Reply:
It looks to me like if the handlers are not passed, then all of the event handlers on the client will be removed for those events. This would mean that event handlers on a client passed into the `FlagProvider` would be reset when the `FlagProvider` is unmounted.